src/layouts/sidebar-sidecar/types.ts

```diff
@@ -18,8 +18,8 @@ interface BaseProps {
 /**
  * `PropsForSidebar` defines the properties that represent `Sidebar` behavior.
  * This approach allows us to require either (not both) `sidebarProps` and
```

Comment:
**Nitpick** ⛏️ 

I'd personally find a list-ish format to be more clear here:

```suggestion
 * This approach allows us to:
 * - allow either `sidebarProps` or `AlternateSidebar` to be passed
 * - allow neither to be passed (in which case the sidebar is not rendered)
 * - cause a type error if both are passed
```

Reply:
Great idea! I'll get this updated, and will clarify a little extra on "in which case the sidebar is not rendered".